cmd/svcat/class/describe_cmd.go

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package class
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/command"
+	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/output"
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	"github.com/spf13/cobra"
+)
+
+type describeCmd struct {
+	*command.Context
+	traverse     bool
+	lookupByUUID bool
+	uuid         string
+	name         string
+}
+
+// NewDescribeCmd builds a "svcat describe class" command
+func NewDescribeCmd(cxt *command.Context) *cobra.Command {
```

Comment:
This is probably another one that we should rename away (in a follow-up, not this PR) from the kubectl verb to something else, maybe inspect?